src/main/java/com/fasttime/domain/review/service/ReviewService.java

```diff
@@ -0,0 +1,101 @@
+package com.fasttime.domain.review.service;
+
+import com.fasttime.domain.member.entity.Member;
+import com.fasttime.domain.member.exception.MemberNotFoundException;
+import com.fasttime.domain.member.repository.MemberRepository;
+import com.fasttime.domain.review.dto.request.ReviewRequestDTO;
+import com.fasttime.domain.review.entity.Review;
+import com.fasttime.domain.review.entity.ReviewTag;
+import com.fasttime.domain.review.entity.Tag;
+import com.fasttime.domain.review.exception.ReviewAlreadyDeletedException;
+import com.fasttime.domain.review.exception.ReviewAlreadyExistsException;
+import com.fasttime.domain.review.exception.ReviewNotFoundException;
+import com.fasttime.domain.review.exception.TagNotFoundException;
+import com.fasttime.domain.review.exception.UnauthorizedAccessException;
+import com.fasttime.domain.review.repository.ReviewRepository;
+import com.fasttime.domain.review.repository.TagRepository;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class ReviewService {
+
+    private final ReviewRepository reviewRepository;
+    private final TagRepository tagRepository;
+    private final MemberRepository memberRepository;
+
+    public Review createReview(ReviewRequestDTO requestDTO, Long memberId) {
+        Member member = memberRepository.findById(memberId)
+            .orElseThrow(() -> new MemberNotFoundException());
```

Comment:
알고 계시겠지만 혹시 도움이 될까 적어봅니다. 이렇게 표현하는 것도 방법도 있으니, 참고만 해주세요~😊
```suggestion
            .orElseThrow(MemberNotFoundException::new);
```

Reply:
확인했습니다! 알려주셔서 감사합니다 ㅎㅎ 😊😊